klippy/extras/replicape.py

```diff
@@ -134,7 +141,8 @@ def __init__(self, config):
             "power_e": (pca9685_pwm, 5), "power_h": (pca9685_pwm, 3),
             "power_hotbed": (pca9685_pwm, 4),
             "power_fan0": (pca9685_pwm, 7), "power_fan1": (pca9685_pwm, 8),
-            "power_fan2": (pca9685_pwm, 9), "power_fan3": (pca9685_pwm, 10) }
+            "power_fan2": (pca9685_pwm, 9), "power_fan3": (pca9685_pwm, 10),
+            "P9_14": (linux_pwm, 0), "P9_16": (linux_pwm, 1) }
```

Comment:
If you expand the self.pins definition, I think it would also be necessary to update the self.enabled_channels setting as well.

-Kevin

Reply:
the channels are only for the pca9685